src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsEFEMKernel.hpp

```diff
@@ -95,7 +95,7 @@ class SinglePhase :
                CRSMatrixView< real64, globalIndex const > const & inputMatrix,
                arrayView1d< real64 > const & inputRhs,
                real64 const (&inputGravityVector)[3],
-               arrayView1d< string const > const & fluidModelNames ):
+               string const & fluidModelKey ):
```

Comment:
I thought those had to be passed by value and not by reference.

Reply:
They do!!